packages/admin/admin-date-time/src/timePicker/FinalFormTimePicker.tsx

```diff
@@ -2,7 +2,7 @@ import { FieldRenderProps } from "react-final-form";
 
 import { TimePicker, TimePickerProps } from "./TimePicker";
 
-export type FinalFormTimePickerProps = TimePickerProps & FieldRenderProps<string, HTMLInputElement | HTMLTextAreaElement>;
+type FinalFormTimePickerProps = TimePickerProps & FieldRenderProps<string, HTMLInputElement | HTMLTextAreaElement>;
```

Comment:
This should be added to index.ts with deprecation notice

Reply:
done